libr/include/r_bin.h

```diff
@@ -320,7 +320,6 @@ typedef struct r_bin_t {
 	ut64 filter_rules;
 	bool demanglercmd;
 	bool verbose;
-	bool io_owned;
```

Comment:
@radare i removed this, it wasn't ever used. It was set in r_bin_load but that path was never reached actually...